sample-kotlin/src/main/kotlin/net/helpscout/samples/beacon/kotlin/SampleApplication.kt

```diff
@@ -0,0 +1,24 @@
+package net.helpscout.samples.beacon.kotlin
+
+import android.app.Application
+import com.helpscout.beacon.Beacon
+
+class SampleApplication : Application() {
+
+    // TODO replace with your Beacon Id from Help Scout
+    private val beaconId = ""
```

Comment:
🔍 Check if set and throw otherwise with a meaningful message.

Reply:
I mentioned this above but here's more detail. The SDK already throws a meaningful message `com.helpscout.beacon.SDKInitException: A Beacon ID is required to initialize the library` with the ethos of keeping these samples concise I feel we should defer to the error messaging the SDK rather than adding a specific check for the `private val beaconId = ""` being empty. Sound good? LMK if there's an alterntive you prefer